### PR TITLE
Applied coding standards

### PR DESCRIPTION
**Pull request by Chownas**
Updated code to reflect coding standards

### CODE REVIEW

File: Plugins/OWI_EditorEnhancements/Source/OWI_EditorEnhancements/Private/OWI_EditorEnhancements.cpp [hunk 1]
```diff
@@ -1,7 +1,9 @@
 // Copyright Offworld Industries Ltd. All Rights Reserved.
 
 #include "OWI_EditorEnhancements.h"
+
```
Comment (regner) [inline Plugins/OWI_EditorEnhancements/Source/OWI_EditorEnhancements/Private/OWI_EditorEnhancements.cpp:4]: What code standard of ours requires an extra line break here?

Comment (Mat-Stevenson-OWI) [reply Plugins/OWI_EditorEnhancements/Source/OWI_EditorEnhancements/Private/OWI_EditorEnhancements.cpp:4]: Order of includes is declaration header, engine/thridparty code and then our headers i think. A line break between the categories is expected too.

**Review by Mat-Stevenson-OWI — APPROVED** [review]

File: Plugins/OWI_EditorEnhancements/Source/OWI_EditorEnhancements/Private/OWI_EditorEnhancements.cpp (hunk 1 above)
Comment (Chownas) [reply Plugins/OWI_EditorEnhancements/Source/OWI_EditorEnhancements/Private/OWI_EditorEnhancements.cpp:4]: Separating Engine includes from project includes. OWI_EditorEnhancements is the project include while ContentBrowserModule is an Engine include.

File: Plugins/OWI_EditorEnhancements/Source/OWI_EditorEnhancements/Private/OWI_UnReleasedAssetsHandler.cpp [hunk 2]
```diff
@@ -1,12 +1,13 @@
 // Copyright Offworld Industries Ltd. All Rights Reserved.
 
 #include "OWI_UnReleasedAssetsHandler.h"
-#include "Misc/ConfigCacheIni.h"
-#include "Misc/FileHelper.h"
+
+#include "EditorAssetLibrary.h"
 #include "OWI_ReferenceRulesHandler.h"
```
Comment (regner) [inline Plugins/OWI_EditorEnhancements/Source/OWI_EditorEnhancements/Private/OWI_UnReleasedAssetsHandler.cpp]: Should this include not be moved to the bottom and separated?

Comment (Chownas) [reply Plugins/OWI_EditorEnhancements/Source/OWI_EditorEnhancements/Private/OWI_UnReleasedAssetsHandler.cpp]: You are correct, fixed

**Review by regner — APPROVED** [review]

**Pull request merged**